backend/src/index.ts

```diff
@@ -384,6 +385,17 @@ handlers[API_PROFILE] = async (input) => {
   };
 };
 
+handlers[API_PROFILE_CREATE] = async (input) => {
+  const user = await input.requireAuthedUser();
+  await db.put(`user:${user.id}`, JSON.stringify(user));
```

Comment:
I think we should make `validateJwtAndGetUser` actually read the user (and only put if it doesn't exist). That way when you're updating the profile it doesn't stomp everything here.

Reply:
Cool, it looks a little weird but I think I've handled that here:
https://github.com/TrevorSundberg/madeitforfun/pull/9/files#diff-fc253a477bb07e861a96b7aad84ea5ecR182

Reply:
Looks good!